src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (firstValue) => {
+    count++;
+
+    return (secondValue) => {
+      return count > 3 && count % 2 === 0
+        ? 'Bzzz... Error!'
+        : firstValue + secondValue;
```

Comment:
Try not to combine && and ternary operators.
```suggestion
      const shouldReturnError = count > 3 && count % 2;

      return shouldReturnError === 0
        ? 'Bzzz... Error!'
        : firstValue + secondValue;
```